clicktests/test.generic.js

```diff
@@ -95,9 +95,19 @@ suite.test('Test showing commit side by side diff between two commits', function
   helpers.click(page, '[data-ta-clickable="commit-sideBySideDiff"]');
   helpers.waitForElementVisible(page, '[data-ta-container="commitLineDiffs"]', function() {
     setTimeout(function() {                           // let it finish making api call
-      helpers.click(page, '[data-ta-clickable="node-clickable-0"]'); // De-select again
+      // helpers.click(page, '[data-ta-clickable="node-clickable-0"]'); // De-select again
```

Comment:
This is because we need to keep "node clicked" state.  if we don't keep the state test will fail as word wrap button will not be visible.